### PR TITLE
updated liclipse (3.3.0)

### PR DESCRIPTION
**Pull request by pachecolp**
#### Editing an existing cask

- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/liclipse.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 cask 'liclipse' do
-  version '3.0.6'
-  sha256 'e5bf65644729267e436ee876cededabd79fedcdadfe43930ea45e72b359c31cf'
+  version '3.3.0'
+  sha256 '918d8a388b093662ba3eb28211dc25d0f83351239666c57e19ab7b5ac484c1a8'
 
-  # googledrive.com/host/0BwwQN8QrgsRpLVlDeHRNemw3S1E was verified as official when first introduced to the cask
-  url "https://googledrive.com/host/0BwwQN8QrgsRpLVlDeHRNemw3S1E/LiClipse%20#{version}/liclipse_#{version}_macosx.cocoa.x86_64.dmg"
+  url "http://www.mediafire.com/file/n4thtt7j07941m8/liclipse_#{version}_macosx.cocoa.x86_64.dmg"
```
Comment (miccal) [inline Casks/liclipse.rb]: You still need a comment above the `url` as it is different from the `homepage`.

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/liclipse.rb (hunk 1 above)
Comment (grootknol) [inline Casks/liclipse.rb]: Actually, I copied it from the homepage.
screen shot 2017-01-20 at 12 21 33

Comment (vitorgalvao) [inline Casks/liclipse.rb]: @grootknol What @miccal means is that the domain is different. It still needs the comment (as you can see by the fact the build is failing).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/liclipse.rb (hunk 1 above)
Comment (grootknol) [inline Casks/liclipse.rb]: Ok, but what should I do? Just add the comment? Is it ok for me to do that, since I was the one introducing the change?

Comment (vitorgalvao) [inline Casks/liclipse.rb]: Yes, it is OK for you to be the one to add the comment. We as maintainers just have to verify if the comment is accurate/truthful.

**Comment by grootknol** [comment]
> Yes, it is OK for you to be the one to add the comment. We as maintainers just have to verify if the comment is accurate/truthful.
> 
Done. Sorry for the misunderstanding.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**